Health/src/main/java/com/sciencesquad/health/MainActivity.java

```diff
@@ -34,6 +43,13 @@ protected void onCreate(Bundle savedInstanceState) {
 
 		NavigationView navigationView = (NavigationView) findViewById(R.id.nav_view);
 		navigationView.setNavigationItemSelectedListener(this);
+
+		RealmConfiguration defaultConfig = new RealmConfiguration.Builder(this)
```

Comment:
Either use a static block:

```java
public class Foo {
    static {
        // only called once ever
    }
    
    // code here...
}
```

Or wait for the Module class to do the initialization procedures and hook into `BaseApplication.AppCreateEvent`...